pallets/portal/src/tests.rs

```diff
@@ -217,6 +220,7 @@ fn cant_submit_without_registering() {
 }
 
 #[test]
+#[ignore]
```

Comment:
why ignores here?

Reply:
We need to generate new testing data, as the fields for the registration have changed. We have a plan a created some tickets for that already.